modules/cooler/makebins/main.nf

```diff
@@ -0,0 +1,33 @@
+process COOLER_MAKEBINS {
+    tag "${cool_bin}"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::cooler=0.8.11" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/cooler:0.8.11--pyh5e36f6f_1':
+        'quay.io/biocontainers/cooler:0.8.11--pyh5e36f6f_1' }"
+
+    input:
+    tuple path(chromsizes), val(cool_bin)
+
+    output:
+    path ("*.bed")       , emit: bed
+    path ("versions.yml"), emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    cooler makebins \\
+        $args \\
+        ${chromsizes} \\
+        ${cool_bin} > cooler_bins_${cool_bin}.bed
```

Comment:
The output file needs to be configurable, as per https://nf-co.re/docs/contributing/modules#naming-conventions